### PR TITLE
upgrade cdash  server  

### PR DESCRIPTION
**Pull request by mer-a-o**
## Description

* This PR updates CDash server to the new docker-based server. 
* CDash url is now read from a text file stored on S3 rather than being hard-coded in CI codes. 
* I will change the CDash url from `cdash.new.jcsda.org` to `cdash.jcsda.org` after updating all the repos. 

### Issue(s) addressed
- related  to https://github.com/JCSDA-internal/jedi-tools/issues/174

## Acceptance Criteria (Definition of Done)
test reports get uploaded to the new CDash server.

## Dependencies
None

## Impact
None

### CODE REVIEW

File: CI/cdash-url.sh [hunk 1]
```diff
@@ -1,8 +1,9 @@
 #!/bin/bash
 
 dir=$1
+cdash_url=$2
 tag=$(head -1 $dir/TAG)
 Done=$(cat $dir/$tag/Done.xml)
 buildID=$(echo $Done | grep -o -P '(?<=buildId>).*(?=</build)')
-URL=https://cdash.jcsda.org/buildSummary.php?buildid=$buildID
+URL=$cdash_url/viewTest.php?buildid=$buildID
```
Comment (climbfuji) [inline CI/cdash-url.sh:8]: Is this change intended? `buildSummary.php` to `viewTest.php`?

Comment (rhoneyager) [reply CI/cdash-url.sh:8]: I prefer the older one, because we do have some compiled C++ and Fortran code in ioda-converters. We want to track build warnings for Ron's development.

Comment (rhoneyager) [inline CI/cdash-url.sh:8]:
```suggestion
URL=$cdash_url/buildSummary.php?buildid=$buildID
```

Comment (mer-a-o) [reply CI/cdash-url.sh]: It should  be `buildSummary`.

**Review by srherbener — APPROVED** [review]
Looks good. Thanks!

**Review by rhoneyager — APPROVED** [review]
